### PR TITLE
Initial documentation update

### PR DESCRIPTION
**Pull request by chrisseaton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,81 +1,102 @@
...
 
-```
-$ rvm install jruby
-```
+##### Do you run Rails?
 
-You can also [download packages from the JRuby website](http://jruby.org/download) that
-you can unpack and run in place.
+We do run Rails, and pass the majority of the Rails test suite. But we are
+missing support for OpenSSL and Nokogiri make it not practical to run real Rails
```
Comment (eregon) [inline README.md]: `which` make it ...

Comment (eregon) [inline README.md]: I prefer `unpractical` but do as you feel is best

File: README.md [hunk 2]
```diff
@@ -1,81 +1,102 @@
...
+applications at the moment.
 
-## Building JRuby from source
+##### What is happening with AOT, startup time, and the SubstrateVM
```
Comment (eregon) [inline README.md]: + `?`

**Comment by chrisseaton** [comment]
Fixed.

**Comment by pitr-ch** [comment]
👍

**Pull request merged**